pkg/operator/starter.go

```diff
@@ -65,6 +67,13 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
+	// operator.openshift.io client, used for ClusterCSIDriver
+	operatorClientSet, err := opclient.NewForConfig(controllerConfig.KubeConfig)
+	if err != nil {
+		return err
+	}
+	operatorInformers := opinformers.NewSharedInformerFactory(operatorClientSet, time.Minute)
```

Comment:
1 minute resync seems kind of agressive for an object that typically doesn't change, but the alternative is that a change to StorageClassState could take a long time to take effect.